docs/SCD2-sparkcompute.md

```diff
@@ -56,6 +58,26 @@ After deduplicating, the resulting records will be:
 
 **Blacklist:** Blacklist for fields to ignore to compare when deduplicating the record.
 
+**Preserve Target** Preserve all the records from the scd2 target table when deduplicating.
+
+**Target Field** The name of the field that tells whether the record is coming from the scd2 target table. This field must be boolean.
```

Comment:
This is only used if preserve target is set to true. Can you add a filter to the widget so this is hidden unless preserve = true?

See https://github.com/data-integrations/http/blob/develop/widgets/HTTP-batchsource.json#L501 for some examples

See https://github.com/data-integrations/http/blob/develop/widgets/HTTP-batchsource.json#L501 for some examples